app/layout.tsx

```diff
@@ -1,3 +1,4 @@
+import { MainLayout } from '@/components/layout/main-layout';
 import './globals.css';
 import type { Metadata } from 'next';
 import { Inter } from 'next/font/google';
```

Comment:
We still have a the inter font which currently is wrong. Ideally we would remove this one and instead of inter import poppins https://nextjs.org/docs/pages/building-your-application/optimizing/fonts

Reply:
@malinovic fixed